Doc/distutils/index.rst

```diff
@@ -12,6 +12,13 @@
    :ref:`distributing-index`
       The up to date module distribution documentations
 
+.. note::
+
+   The ``distutils`` namespace has been deprecated and will be removed in
```

Comment:
I wonder if «namespace» will be clear to all readers — what do you think of saying `distutils package` or `distutils modules`?

Reply:
I went back and forth on all of those, including during the PEP discussions.

"The entire `distutils` package" is about the only other one I'd be happy with, as people really wanted to argue for only deprecating parts of it (even though most of the bits they wanted to "save" already exist elsewhere in the stdlib or supported packages).

Reply:
`entire distutils package` sounds good to me.